contracts/seed/Seed.sol

```diff
@@ -30,7 +30,7 @@ contract Seed {
     uint256 public hardCap;
     uint256 public seedAmountRequired; // Amount of seed required for distribution
     uint256 public feeAmountRequired; // Amount of seed required for fee
-    uint256 public price; // price of a SeedToken, expressed in fundingTokens, with precision of 10**18
+    uint256 public price; // price of a SeedToken, expressed in fundingTokens, in units of funding token
```

Comment:
can we be more thorough:  "in units of funding token with precision of 10\*\*funding token decimals" (typically 10**18)?